lib/rules/disallow-identifier-names.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Disallows a specified set of identifier names.
+ *
+ * Type: `Array`
+ *
+ * Values: Array of strings, which should be disallowed as identifier names
+ *
+ * #### Example
+ *
+ * ```js
+ * "disallowIdentifierNames": ['temp', 'foo']
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * var good = 1;
+ * object[fine] = 2;
+ * object.fine = 3;
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * var temp = 1;
+ * object[foo] = 2;
```

Comment:
The rule currently does not prohibit ```object['foo']``` (therefore, the foo should not have quotes in the example).

That said, I agree that we want to prohibit ```object['foo']```, too. Let me implement it.